### PR TITLE
Austin/fuzzywuzzy

### PR DESCRIPTION
**Pull request by AAAstorga**
fixed minor issues according to paper doc and added fuzzy search 👍

### CODE REVIEW

File: TCAT/Controllers/SearchResultsTableViewController.swift [hunk 1]
```diff
@@ -107,6 +107,8 @@ class SearchResultsTableViewController: UITableViewController, UISearchBarDelega
 //        initialTableViewIndexMinY = tableViewIndexController.tableViewIndex.indexRect().minY
 //        setUpIndexBar(contentOffsetY: 0.0)
         sections = createSections()
+        searchBar?.becomeFirstResponder()
+        searchBar?.tintColor = .black
```
Comment (mattbarker016) [inline TCAT/Controllers/SearchResultsTableViewController.swift]: use a color in UIColor+shared

File: TCAT/Utilities/SearchTableViewHelpers.swift [hunk 2]
```diff
@@ -102,9 +102,10 @@ func sectionIndexesForBusStop() -> [String: Int] {
 func parseGoogleJSON(searchText: String, json: JSON) -> Section {
     var itemTypes: [ItemType] = []
     let filteredBusStops = getAllBusStops().filter({(item: BusStop) -> Bool in
-        let stringMatch = item.name.lowercased().range(of: searchText.lowercased())
-        return stringMatch != nil
+        let levenshteinScore = String.fuzzPartialRatio(str1: item.name.lowercased(), str2: searchText.lowercased())
+        return levenshteinScore > 75
```
Comment (mattbarker016) [inline TCAT/Utilities/SearchTableViewHelpers.swift]: create a `constant` variable to abstract magic number from code, possible as variable in struct or enclosing class

**Review by mattbarker016 — COMMENTED** [review]
Mostly good to go!

File: TCAT/Controllers/SearchResultsTableViewController.swift (hunk 1 above)
Comment (AAAstorga) [reply TCAT/Controllers/SearchResultsTableViewController.swift]: black is not in UIColor+shared

**Pull request merged**